src/App.js

```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
+import { Routes, Route } from 'react-router-dom';
 import './App.css';
+import Header from './components/navigation/Header';
+import Books from './pages/books';
+import Categories from './components/category/Categories';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use `Arrow function` on your main `App` component for best practices. I suggest you apply this approach in all your components